dist/index.js

```diff
@@ -580,10 +587,17 @@ async function getTeamPingIssues (octokit, org, team, authors, commenters, since
   // Add the created since date query
   query = query.concat(`+created%3A%3E${since}`)
 
-  // Add ignore repos query
-  ignoreRepos.forEach(elem => {
-    query = query.concat(`+-repo%3A${elem}`)
-  })
+  if (includeRepos.length > 0) {
```

Comment:
I've forgotten so much JS, if this is null and we call don't `.length` is that alright? Or do we know this will always be definied

Reply:
I believe when we initialize the variable [here](https://github.com/rachmari/first-responder/pull/14/files#diff-3d2b59189eeedc2d428ddd632e97658fe310f587f7cb63b01f9b98ffc11c0197R513), it will either be an empty array or an array with values. So I don't think we need to worry about a nil check here

Reply:
The `includerepos` and `ignoreRepos` variables are set up at the top to empty arrays if the input isn't defined: https://github.com/rachmari/first-responder/pull/14/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R15

So checking the length is 👍.